homeassistant/components/august/camera.py

```diff
@@ -64,12 +64,14 @@ def model(self):
 
         if self._image_url is not latest.image_url:
             self._image_url = latest.image_url
-            self._image_content = requests.get(
-                self._image_url, timeout=self._timeout
-            ).content
+            self._image_content = await self.hass.async_add_job(self._camera_image)
```

Comment:
Use `hass.async_add_executor_job` instead of `hass.async_add_job`. The latter is legacy for general use.

Reply:
Will change